src/test/java/it/pagopa/pn/downtime/DocumentCompositionTest.java

```diff
@@ -0,0 +1,73 @@
+package it.pagopa.pn.downtime;
+
+import com.openhtmltopdf.pdfboxout.PdfRendererBuilder;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DocumentCompositionTest {
```

Comment:
I'm not sure about this solution. I have some questions:
1. Why we redefine the html2Pdf method?
2. Why we test the DocumentComposition and not the LegalFactGenerator?
3. Can we do the same job done for delivery-push? check the LegalFactGeneratorTest file